draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -70,14 +70,16 @@ This document lists some use cases that demonstrate the need for a mechanism to
 
 Wireless networks inherently experience large variations in link capacity due to several factors.
 These include the change in wireless channel conditions, interference between proximate cells and channels or because of the end user movement.
-These variations in link capacity can be in the order of a millisecond or less.
-Media frames may experience high sojourn times in the wireless network or the flow may experience lower throughput, neither of which is desirable.
+These variations in link capacity can be in the order of a millisecond or less which is far lower than RTT.
```

Comment:
seems "link capacity" is not consistent with "millisecond" mention.

Reply:
Since the scheduling interval for 5G mid band is 0.5ms, there can be variations in that interval.
With mmWave, the scheduling interval is 0.0125 ms and much larger variations in capacity over that period of time.
Both number of users and change the wireless medium result in large link variations in sub-ms timeframes.
The argument was intended to say that rate estimation in CCA take multiple RTTs and that is several orders of magnitude slower than the change of radio link capacity.

If the above reasoning is ok, I'll re-write that sentence to be more clear :
"Wireless link capacity can vary in sub-millisecond timeframes while rate estimation at the server (sender) takes multiple RTTs, or several tens of milliseconds."

Reply:
Just to clarify the scheduling interval (0,5 ms, or 0.0125 ms) is the minimum period. The scheduler may allocate 10 or more slots to a user (less if there are more users). So the variation in capacity available for a user can be seen at 0.0125*10 onwards ~ 0.1ms for mmWave (and around 2-3 ms in congested 5G mid-band)

Maybe better wording would be:
"Wireless link capacity can vary in the order of a millisecond or less while CCA estimates of rate at the server (sender) takes multiple RTTs, or several tens of milliseconds".